scripts/ci-test/build_changed.ts

```diff
@@ -60,9 +59,13 @@ if (!allTestConfigNames.includes(inputTestConfigName)) {
 
 const config = testConfig[inputTestConfigName]!;
 
-buildForTests(config, buildAppExp);
+buildForTests(config, buildAppExp, buildAppCompat);
 
-async function buildForTests(config: TestConfig, buildAppExp = false) {
+async function buildForTests(
+  config: TestConfig,
+  buildAppExp = false,
+  buildAppCompat = false
```

Comment:
Was the script broken because of the missing argument until now? but the workflow seems to work fine?

Reply:
It used to be a global variable so it worked but maybe in an unexpected way.